ui/lib/apps/SlowQuery/utils/tableColumns.tsx

```diff
@@ -24,97 +28,47 @@ function ResultStatusBadge({ status }: { status: 'success' | 'error' }) {
   )
 }
 
-function commonColumnName(fieldName: string): any {
-  return <TextWithInfo.TransKey transKey={`slow_query.fields.${fieldName}`} />
-}
+//////////////////////////////////////////
+// Notice:
+// The key field value in the following methods is case-sensitive
+// They should keep the same as the column name in the slow query table
+// Ref: pkg/apiserver/slowquery/queries.go SlowQuery struct
+const TRANS_KEY_PREFIX = 'slow_query.fields'
 
 function sqlColumn(
   _rows?: { query?: string }[], // used for type check only
   showFullSQL?: boolean
 ): IColumn {
-  return {
-    name: commonColumnName('sql'),
-    key: 'sql',
-    fieldName: 'sql',
-    minWidth: 200,
-    maxWidth: 500,
-    onRender: (rec) =>
-      showFullSQL ? (
-        <TextWrap multiline>
-          <HighlightSQL sql={rec.query} />
-        </TextWrap>
-      ) : (
-        <Tooltip
-          title={<HighlightSQL sql={rec.query} theme="dark" />}
-          placement="right"
-        >
-          <TextWrap>
-            <HighlightSQL sql={rec.query} compact />
-          </TextWrap>
-        </Tooltip>
-      ),
-  }
+  const column = sqlTextColumn(TRANS_KEY_PREFIX, 'Query', showFullSQL)
+  column.name = commonColumnName(TRANS_KEY_PREFIX, 'sql')
+  return column
 }
 
-function digestColumn(
-  _rows?: { digest?: string }[] // used for type check only
+function connectionIDColumn(
+  _rows?: { connection_id?: number }[] // used for type check only
 ): IColumn {
   return {
-    name: commonColumnName('digest'),
-    key: 'Digest',
-    fieldName: 'digest',
+    name: commonColumnName(TRANS_KEY_PREFIX, 'connection_id'),
+    key: 'Conn_ID',
+    fieldName: 'connection_id',
     minWidth: 100,
-    maxWidth: 150,
-    onRender: (rec) => (
-      <Tooltip title={rec.digest}>
-        <TextWrap>{rec.digest}</TextWrap>
-      </Tooltip>
-    ),
+    maxWidth: 120,
   }
 }
 
-function instanceColumn(
-  _rows?: { instance?: string }[] // used for type check only
+function _timestampColumn(
```

Comment:
Why do we have a very different name style for this one?

Reply:
Because it calls `timestampColumn()` method which is extracted as a re-used method... maybe I can choose another name.